### PR TITLE
Audit: attach organization

### PR DESCRIPTION
**Pull request by stsewd**
Realized that after a project is deleted,
we don't know from what organization it was.
(well, we can with the historical models,
but having that information here makes it more easy to filter by).

### CODE REVIEW

File: readthedocs/audit/models.py [hunk 1]
```diff
@@ -117,6 +117,27 @@ class AuditLog(TimeStampedModel):
...
+        null=True,
+        db_index=True,
+        on_delete=models.SET_NULL,
+    )
+    log_organization_id = models.IntegerField(
+        _('Organization ID'),
+        blank=True,
+        null=True,
+        db_index=True,
+    )
```
Comment (humitos) [inline readthedocs/audit/models.py:132]: Why is the organization id relevant? Shouldn't be enough with the organization slug?

File: readthedocs/audit/models.py [hunk 2]
```diff
@@ -157,6 +178,11 @@ def save(self, **kwargs):
         if self.project:
             self.log_project_id = self.project.id
             self.log_project_slug = self.project.slug
+            organization = self.project.organizations.first()
+            if organization:
```
Comment (humitos) [inline readthedocs/audit/models.py:182]: Un-related to this PR: now that we have Organizations in .org, we can migrate the `ManyToMany` from Organization model into a ForeignKey in the Project model and avoid all these `.first()` checks we have all around the code.

**Review by humitos — APPROVED** [review]

File: readthedocs/audit/models.py (hunk 1 above)
Comment (stsewd) [reply readthedocs/audit/models.py:132]: We want to keep track of the organization after it has been deleted. The id is the source of truth, as another organization can be created with the same slug. We save the slug to make it more friendly to recognize the original organization or display.

Comment (ericholscher) [reply readthedocs/audit/models.py:132]: This should be a comment in the code, not in a PR ;)

**Pull request merged**